fr24-sharing-key-generator/entrypoint.sh

```
@@ -0,0 +1,113 @@
+EOF
+
+echo "[$(date)] Starting nginx..."
+nginx -g 'daemon off;' &
+NGINX_PID=$!
+
+echo "[$(date)] Running fr24feed signup wizard via expect..."
+
+# Append expect output to log and show in stdout
+/signup.exp 2>&1 | tee -a /usr/share/nginx/html/index.html
+
+SIGNUP_EXIT_CODE=${PIPESTATUS:-${?}}
```

Shellcheck (0.10.0)

[warning] In POSIX sh, PIPESTATUS is undefined.

(SC3028)